sky/provision/kubernetes/config.py

```diff
@@ -446,17 +521,15 @@ def _configure_fuse_mounting(provider_config: Dict[str, Any]) -> None:
     which exposes the host /dev/fuse device as a Kubernetes resource. The
     SkyPilot pod requests this resource to mount the FUSE filesystem.
 
-    We create this daemonset in a common namespace, which is configurable in the
-    provider config. This allows the FUSE mounting sidecar to be shared across
-    multiple tenants. The default namespace is 'sky-system' (populated in
-    clouds.Kubernetes)
+    We create this daemonset in the skypilot_system_namespace, which is
+    configurable in the provider config. This allows the FUSE mounting sidecar
+    to be shared across multiple tenants. The default namespace is
+    'sky-system' (populated in clouds.Kubernetes).
```

Comment:
it seems we are using `skypilot-system` instead, should we update the comment here?
```suggestion
    'skypilot-system' (populated in clouds.Kubernetes).
```